tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -62,4 +65,90 @@ describe('Test Fleet access with RBAC with custom roles', { tags: '@rbac' }, ()
     })
   )
 
+  qase(43,
+    it('Test "Std-Base" user with custom role "list" and "create" to "fleetworkspaces" can list and create workspaces but NOT  "edit" nor "delete" them', { tags: '@fleet-43' }, () => {
+      
+      const stduser = "std-user-43"
+      const customRoleName = "fleetListAndCreateRole"
+
+      //  Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+      
+      cy.createRoleTemplate({
+        roleType: "Global",
+        roleName: customRoleName,
+        rules: [
+          { resource: "fleetworkspaces", verbs: ["list", "create"]},
+          { resource: "gitrepos", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+          { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        ]
+      });
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName)
+
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // What the user should be able to access
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.wait(500)
+      cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Bundles');
+      cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Workspaces');
+      cy.verifyTableRow(0, 'Active', 'fleet-default');
+      cy.verifyTableRow(1, 'Active', 'fleet-local');
+      cy.get('a.btn.role-primary').contains('Create').should('be.visible');
+      
+      // What the user should NOT be able to access
```

Comment:
// What `verbs` the user should NOT be able to access <-- Please add highlighted word for better understanding.

Reply:
This applies to all places where this comment it used.

Reply:
Verbs not to to be able to access are specified on the title of the test.
I added this comment for clearness where to focus. I could remove it.
I think is redundant to point again the verbs mentioned in title, but I will if desired

Reply:
Please, don't remove it. They are good. I know test is using verbs but here clearly mentioning about the access to the specified menu by segregating with verbs would be much clearer, IMO :smile: 
Feel free to keep it as it is no worries :smile:

Reply:
Changed to:
```
// Ensuring the user is not able to "edit" or "delete" workspaces.
```

Reply:
Fine with that Thank You :smile: